rpcd/playbooks/roles/elasticsearch/tasks/upgrade_pre_flight.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Install Pre Flight Checks
+  template:
+    src: upgrade-pre-flight-checks.py.j2
+    dest: /opt/upgrade-pre-flight-checks.py
+    owner: root
+    group: root
+    mode: 0755
+  delegate_to: "{{ groups['elasticsearch_all'][0] }}"
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - es-pre-flight
+
+- name: Run Pre Flight Checks
+  shell: /opt/upgrade-pre-flight-checks.py
+  delegate_to: "{{ groups['elasticsearch_all'][0] }}"
+  register: pre_flight_status
```

Comment:
This old comment showed back up because you readded `register: pre_flight_status`.

To future reviewers, this is no longer a valid conversation, but y'know, GitHub thinks it is.